File: Python-Task-03/Python/number_analysis.py
n = int(input("Enter a number: "))

total = 0
even_count = 0
odd_count = 0

for i in range(1, n + 1):
    total += i

    if i % 2 == 0:
        even_count += 1
    else:
        odd_count += 1

print("Sum =", total)
print("Even Numbers =", even_count)
print("Odd Numbers =", odd_count)

File: Python-Task-03/C/number_analysis.c
#include <stdio.h>

int main() {
    int n;
    int sum = 0;
    int even = 0;
    int odd = 0;

    printf("Enter a number: ");
    scanf("%d", &n);

    for(int i = 1; i <= n; i++) {
        sum += i;

        if(i % 2 == 0)
            even++;
        else
            odd++;
    }

    printf("Sum = %d\n", sum);
    printf("Even Numbers = %d\n", even);
    printf("Odd Numbers = %d\n", odd);

    return 0;
}